fastai/vision/data.py

```diff
@@ -11,8 +11,18 @@
 
 # Cell
 @delegates(subplots)
-def get_grid(n, nrows=None, ncols=None, add_vert=0, figsize=None, double=False, title=None, return_fig=False,
-             flatten=True, **kwargs):
+def get_grid(
+    n: int,                 # Number of axes in the returned grid
+    nrows: int=None,        # Number of rows in the returned grid, defaulting to `int(math.sqrt(n))`
+    ncols: int=None,        # Number of columns in the returned grid, defaulting to `ceil(n/rows)`
+    add_vert=0,
+    figsize: tuple=None,    # Width, height in inches of the returned figure
+    double=False,           # Whether to double the number of columns and `n`
+    title: str=None,        # If passed, title set to the figure
+    return_fig: bool=False, # Whether to return the figure created by `subplots`
+    flatten: bool=True,     # Whether to flatten the matplot axes such that they can be iterated over with a single loop
+    **kwargs,               # Keyword argurments passsed to `subplots`
```

Comment:
```suggestion
    **kwargs,
```

Reply:
I just noticed this. I will erase the **kwargs docs and commit again right now